api/v1alpha1/perconaservermysql_types.go

```diff
@@ -351,6 +353,10 @@ type ServiceExposeTogglable struct {
 
 type StatefulAppState string
 
+func (s StatefulAppState) String() string {
+	return cases.Title(language.English).String(string(s))
```

Comment:
what's the advantage of using these over [`strings.ToTitle`](https://pkg.go.dev/strings#ToTitle)?

Reply:
strings.ToTile is deprecated.

Reply:
`strings.Title` is deprecated but there's `strings.ToTitle`

Reply:
Oh nice, I missed it.

Reply:
`strings.ToTitle` for some strange reason returns all capital letters, the same way like `ToUpper` does. 
`cases.Title` is the way to here.